README.md

```diff
@@ -1,3 +1,6 @@
+## Deploy to heroku
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/Dawnfrosty/Mike-bot)
+
```

Comment:
I would suggest removing this and closing the PR.
This doesn't add any information for this repo.

```suggestion
```